content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,225 @@
+weight: 10
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and Sidecar Injector. By default,
+Galley and Sidecar Injector manage their own webhook configurations, which from the
+security perspective is not recommended because a compromised webhook may conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of Galley and Sidecar Injector, to
+manage the webhook configurations of Galley and Sidecar Injector.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed. In the installation,
+[`global.operatorManageWebhooks`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) should
+be set to `true`, and [DNS certificates should be configured](/docs/tasks/security/dns-cert).
+Istio installation guides can be found [here](/docs/setup/install).
+
+* Install [`jq`](https://stedolan.github.io/jq/) for JSON parsing.
+
+## Check webhook certificates
+
+To display the DNS names in the webhook certificates of Galley and Sidecar Injector, run the following commands:
+
+{{< text bash >}}
+$ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+$ kubectl get secret dns.istio-sidecar-injector-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+{{< /text >}}
+
+The output from the above commands should include the DNS names of Galley and Sidecar Injector, respectively:
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-galley.istio-system.svc, DNS:istio-galley.istio-system
+{{< /text >}}
+
+{{< text plain >}}
+X509v3 Subject Alternative Name:
+  DNS:istio-sidecar-injector.istio-system.svc, DNS:istio-sidecar-injector.istio-system
+{{< /text >}}
+
+## `istioctl` enables webhook configurations
+
+1.  Generate `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` by running the following
+command. The YAML file [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml)
+contains an example DNS certificate configuration (details in [the certificate guide](/docs/tasks/security/dns-cert)).
+
+    {{< text bash >}}
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
+        install/kubernetes/helm/istio > istio-webhook-config.yaml
+    {{< /text >}}
+
+<!-- TODO (lei-tang): improve the UX for obtain MutatingWebhookConfiguration -->
+1.  From `istio-webhook-config.yaml`, search `'kind: MutatingWebhookConfiguration'` and save
+the `MutatingWebhookConfiguration` of Sidecar Injector to `sidecar-injector-webhook.yaml`. The following
+is a `MutatingWebhookConfiguration` in an example `istio-webhook-config.yaml`.
```

Comment:
Why not just copy the following two blocks of yaml configs instead?

Reply:
Because a user may want to customize the namespace, etc in the yaml config, the two blocks of yaml configs are not copied.

Reply:
But as far as I see, you didn't tell them to how to customize, the `helm template` command generates a yaml file that contains the config which doesn't seem to be different from the two blocks you gave in the following. So if you want users to customize, why not explicitly mentioning it in the following? We were doing this for other tasks.

Reply:
Added an explanation that the command in the guide uses the default Istio configuration plus the DNS certificate configuration in `values-istio-dns-cert.yaml`, as well as a link to the helm template document. This guide does not require users to customize the namespace, and etc, but the namespace and etc in the validatingwebhookconfiguration and mutatingwebhookconfiguration should be consistent with the users' Istio installation. Therefore, the guide does not provide two hard-coded blocks of yaml configs of validatingwebhookconfiguration and mutatingwebhookconfiguration to copy and paste.

Reply:
Please ensure that the actions the reader needs to take are clear. I was unable to discern all actions needed here.